### PR TITLE
`@cds.collate` annotation

### PR DESCRIPTION
**Pull request by MattSchur**

### CODE REVIEW

**Review by agoerler — APPROVED** [review]
Looks good to me. 

There's also [this location](guides/db-feature-comparison#locale-dependent-sorting), which could benefit from some elaboration.

File: java/persistence-services.md [hunk 1]
```diff
@@ -18,14 +18,30 @@ uacp: Used as link target from Help Portal at https://help.sap.com/products/BTP/
...
+```cds
+entity Books : cuid {
+    title        : localized String(111);
+    descr        : localized String(1111);
+    @cds.collate : false // [!code focus]
+    isbn         : String(40);  // does not require locale-specific sorting // [!code focus]
+}
+```
+
+To disable statement-wide collation for all queries, set [`cds.sql.hana.ignoreLocale`](../java/development/properties#cds-sql-hana-ignoreLocale) to `true`.
```
Comment (renejeglinsky) [inline java/persistence-services.md:44]: Could also be a tip.
```suggestion
:::tip Disable Statement-Wide Collation
To disable statement-wide collation for all queries, set [`cds.sql.hana.ignoreLocale`](../java/development/properties#cds-sql-hana-ignoreLocale) to `true`.
:::
```

File: java/persistence-services.md [hunk 2]
```diff
@@ -18,14 +18,30 @@ uacp: Used as link target from Help Portal at https://help.sap.com/products/BTP/
 
 ## Database Support { #database-support}
 
-The CAP Java SDK has built-in support for various databases. This section describes the different databases and any differences between them with respect to CAP features. There's out of the box support for SAP HANA with CAP currently as well as H2 and SQLite. However, it's important to note that H2 and SQLite aren't an enterprise grade database and are recommended for nonproductive use like local development or CI tests only. PostgreSQL is supported in addition, but has various limitations in comparison to SAP HANA, most notably in the area of schema evolution.
+CAP Java has built-in support for various databases. This section describes the different databases and any differences between them with respect to CAP features. There's out of the box support for SAP HANA with CAP currently as well as H2 and SQLite. However, it's important to note that H2 and SQLite aren't an enterprise grade database and are recommended for nonproductive use like local development or CI tests only. PostgreSQL is supported in addition, but has various limitations in comparison to SAP HANA, most notably in the area of schema evolution.
```
Comment (renejeglinsky) [inline java/persistence-services.md:21]: Is there a difference between CAP Java SDK and CAP Java?

Comment (MattSchur) [reply java/persistence-services.md:21]: No, we use both terms interchangeably

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**